src/conditional-flow/numeric-conditions.js

```diff
@@ -4,21 +4,34 @@
 function isArrayEmpty (array) {
 
   // TODO: write code in this function body to pass the tests
-
+  return array.length == 0 ? true : false;
 }
 
 // This function should return true if num1 is greater than num2, false otherwise
 function isGreaterThan (num1, num2) {
 
   // TODO: write code in this function body to pass the tests
-
+  if (num1 > num2) {
```

Comment:
`num1 > num2` creates a boolean value. So like above with the comments about ternary statements; you can just return this value.

```js
return num1 > num2
```